virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -71,13 +93,19 @@ def get_non_root_disk_names(session, ignore_status=False):
     if s:
         raise exceptions.TestError("Couldn't list block devices: '%s'" % o)
     LOG.debug("lsblk output:\n%s", o)
+
+    # search for additional root disk in case of image_mode
+    root_mountpoints = ["/"]
```

Comment:
if like sebas suggested below,  we do not need to check is_image_mode(), right?
`if mpoint in ["/", "/sysroot"]:`